client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SortBasedShuffleWriter.java

```diff
@@ -377,7 +405,7 @@ private void updateMapStatus() {
   @Override
   public Option<MapStatus> stop(boolean success) {
     try {
-      taskContext.taskMetrics().incPeakExecutionMemory(peakMemoryUsedBytes);
+      taskContext.taskMetrics().incPeakExecutionMemory(getPeakMemoryUsedBytes());
```

Comment:
The variable `private long peakMemoryUsedBytes = 0;` defined in this class could be deleted then?

Reply:
> The variable `private long peakMemoryUsedBytes = 0;` defined in this class could be deleted then?

No, still need

Reply:
I see